File: CodeFestival/Q31-Q40/Q36_구구단_출력하기.js
const n = prompt("숫자를 입력하세요.");
let arr = [];
for (let i=0; i<9; i++) {
    arr[i] = n * (i+1);
}

arr = arr.join(' ');
console.log(arr);

// const num = prompt('1 ~ 9까지의 숫자 중 하나를 입력하세요.')
// let result = '';

// for (let i=1; i<=9; i++){
//   result += i*num + ' ';
// }

// console.log(result);

File: CodeFestival/Q31-Q40/Q36_구구단_출력하기.py
n = int(input())
arr = []
for i in range(0, 9, 1):
    m = n * (i+1)
    arr.append(str(m))

arr = ' '.join(arr)
print(arr)

# n = int(input())
# for i in range(1, 10):
# 	print(n*i, end = ' ')